src/replicache.ts

```diff
@@ -233,6 +233,8 @@ export default class Replicache implements ReadTransaction {
   }
 
   private async _sync(): Promise<void> {
+    var online = true;
```

Comment:
[cbb] no `var` please

Reply:
Looks like one of us fixed this.

Reply:
I'm going to spend a fe minutes tweaking eslint to check for this and also check sample/**